ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -225,16 +229,21 @@ export default class AlertTableRow extends React.Component {
           </FormGroup>
         </td>
         <td className="px-0">
-          <span
-            className={starred ? 'visible' : ''}
+          <button
```

Comment:
I think we'd want to give this an `aria-label` for whether it's "starred" or not.

Reply:
That sounds good.